README.md

```diff
@@ -18,6 +18,8 @@ It is planned to include this library in a future version of the Modelica Standa
 
 ## Current release
 
+Coming release: [Modelica_Synchronous v0.92.2 (2019-03-20?)](../../releases/tag/v0.92.2)
+
```

Comment:
You should wait with adding a link as it will just be dead for now. So better add that when you know that it will be tagged.

Reply:
Ok. That has now been updated.

Reply:
Ok for you?

Reply:
Well what is the purpose of the question mark?